feat(apps): graduate getUserRoomIds experiment to the user bridge (#37547)
RocketChat/Rocket.Chat: .changeset/calm-falcons-gather.md

---
"@rocket.chat/apps-engine": minor
"@rocket.chat/meteor": minor
---

> Graduates the `getUserRoomIds` accessor from the experimental bridge to the stable user bridge after the experiment proved successful so apps can rely on the API under the standard user read permission.

Reviewer: ```suggestion
Adds the `getUserRoomIds` method to the `UserRead` accessor in the Apps-Engine, graduating it from the experimental bridge to the stable user bridge.
```
